### PR TITLE
--fix load data cannot work with relative path issue#708

### PR DESCRIPTION
**Pull request by broccoliSpicy**
**What type of PR is this?**

- [ ] API-change
- [ ] BUG
- [ ] Improvement
- [ ] Documentation
- [ ] Feature
- [ ] Test and CI
- [ ] Code Refactoring

**Which issue(s) this PR fixes:**

issue #708 

**What this PR does / why we need it:**

Not Available

**Special notes for your reviewer:**

Not Available

**Additional documentation (e.g. design docs, usage docs, etc.):**

Not Available

### CODE REVIEW

File: pkg/sql/tree/transformer.go [hunk 1]
```diff
@@ -1934,7 +1935,9 @@ func transformLoadDataStmtToLoad(lds *ast.LoadDataStmt) *Load {
 			ues[i] = transformAssignmentToUpdateExpr(a)
 		}
 	}
-
+	if !filepath.IsAbs(lds.Path) {
```
Comment (daviszhen) [inline pkg/sql/tree/transformer.go:1938]: It is suitable to move the ABS check into the function handleLoadData. In the future, the transformers will be removed after the new parser accomplished.

**Comment by daviszhen** [comment]
Thank you @broccoliSpicy

**Comment by broccoliSpicy** [comment]
oh, I didn't know that the transformer is gonna get removed, thank  you @daviszhen . 
As for the build failure on CentOS/x64, I am quite confused since this is just a minor change.

**Comment by daviszhen** [comment]
> oh, I didn't know that the transformer is gonna get removed, thank you @daviszhen . As for the build failure on CentOS/x64, I am quite confused since this is just a minor change.

The case **Test_transformLoadDataStmtToLoad** failed in transformer_test.go:5749

**Pull request closed without merging**